cime_config/config_component.xml

```diff
@@ -151,20 +151,19 @@
     <valid_values>UNSET,1850,2000,hist,ssp119,ssp126,ssp245,ssp370,ssp434,ssp460,ssp534os,ssp585</valid_values>
     <default_value>1850</default_value>
     <values>
-      <value compset="1850_.*_BLOM.*%ECO"   >1850</value>
-      <value compset="2000_.*_BLOM.*%ECO"   >2000</value>
-      <value compset="HIST_.*_BLOM.*%ECO"   >hist</value>
-      <value compset="20TR_.*_BLOM.*%ECO"   >hist</value>
-      <value compset="CPLHIST_.*_BLOM.*%ECO">1850</value>
-      <value compset="SSP119_.*_BLOM.*%ECO" >ssp119</value>
-      <value compset="SSP126_.*_BLOM.*%ECO" >ssp126</value>
-      <value compset="SSP245_.*_BLOM.*%ECO" >ssp245</value>
-      <value compset="SSP370_.*_BLOM.*%ECO" >ssp370</value>
-      <value compset="SSP434_.*_BLOM.*%ECO" >ssp434</value>
-      <value compset="SSP460_.*_BLOM.*%ECO" >ssp460</value>
-      <value compset="SSP534_.*_BLOM.*%ECO" >ssp534os</value>
-      <value compset="SSP585_.*_BLOM.*%ECO" >ssp585</value>
-      <value compset="SSP370.*_BLOM.*%ECO"  >ssp370</value>
+      <value compset="^1850_.*_BLOM.*%ECO"   >1850</value>
+      <value compset="^2000_.*_BLOM.*%ECO"   >2000</value>
+      <value compset="^HIST_.*_BLOM.*%ECO"   >hist</value>
+      <value compset="^20TR_.*_BLOM.*%ECO"   >1850</value>
```

Comment:
This is the CMIP6 OMIP compset for historical hindcast. It's correct tha currently this is not supported, but I guess it will be, so I would suggest to have 'hist' here.